application-xpoll-ui/src/main/resources/XPoll/XPollJSExtension.xml

```diff
@@ -151,6 +151,23 @@
         }
       });
     });
+    $('.xpollSelect').change(function() {
+      var changedElement = $(this);
+      $('.xpollSelect').each(function() {
```

Comment:
This needs a comment because it's not obvious: "Make sure the other proposals don't use the same value (index)."